tests/jest/unitTest/DicomParserUtils.test.ts

```diff
@@ -1,4 +1,3 @@
-// tests/jest/unitTest/DicomParserUtils.unit.test.tsx
 import { parseDicomFile } from "../../../src/components/DicomData/DicomParserUtils";
```

Comment:
```suggestion
import { parseDicomFile } from "@components/DicomData/DicomParserUtils";
```
Maybe give the new path alias a try, a bit nicer than having so many ../

Reply:
hey I saw the path are set up in the tsconfig alreday, But the build seems failing due to the change?